### DISCUSSION TITLE
lib Component type in playground

### DISCUSSION THREAD

**Discussion by nclemeur** [LinusBorg/vue-lib-template#5] [Q&A]
Hello,

I am trying to setup a similar environment and I have tried with this template, but it look like I can't get the types from the lib components to be visibble in VS Code autocomplete in the playground project. So below is the auto-complete I have on the playground component HelloWorld (you can clearly see the :msg property
![image](https://user-images.githubusercontent.com/7132890/234185637-e30cfcf0-74bc-472b-a82b-657c1d087eee.png)
while below is what I have for the MyComponent (from lib)
![image](https://user-images.githubusercontent.com/7132890/234185916-7008c7e4-0ee5-46be-9f2c-f3b65a6c9ed9.png)

Note also the difference in the coloring the HelloWorld and MyComponent tags...

So any idea on how to achieve a better auto-complete?

**Comment by peoray** [comment]
@nclemeur did you ever figure this out?
